system-tests/projects/dev-server-invalid/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "@test-project/vite-ct-function-api",
+  "version": "0.0.0-test",
+  "private": true,
+  "dependencies": {},
+  "devDependencies": {
+    "@cypress/vite-dev-server": "file:../../../npm/vite-dev-server",
+    "@testing-library/cypress": "8.0.2",
```

Comment:
Probably can make this example project just use `cypress.config.js` to keep the deps to a minimum, each project and dep makes the CI run time a little longer.